helpers/check_acme_routes.sh

```diff
@@ -18,9 +22,15 @@ COMMAND=${1:-"help"}
 # Set DRYRUN variable to true to run in dry-run mode
 DRYRUN="${DRYRUN:-"false"}"
 
+# Set DEBUG variable to true, to print echo messages
```

Comment:
this also enables BASH debugging on line `9`, so I'm not sure it makes sense to use the same env variable for echo messages and also bash debugging

Reply:
I change the comment in `Set DEBUG variable to true, to start bash in debug mode`.
I will remove debug echoes.